rust/agama-software/src/model/packages.rs

```diff
@@ -39,7 +39,16 @@ impl Resolvable {
 
 /// Software resolvable type (package or pattern).
 #[derive(
-    Clone, Copy, Debug, Deserialize, Serialize, strum::Display, utoipa::ToSchema, PartialEq,
+    Clone,
+    Copy,
+    Debug,
+    Deserialize,
+    Serialize,
+    strum::Display,
+    utoipa::ToSchema,
+    PartialEq,
+    Eq,
+    Hash,
```

Comment:
NP: I prefer to have entries alphabetically ordered, as it is easier to read. I would apply it to all parts that do not depend on order.